tests_e2e/test_suites/ext_sequencing.yml

```diff
@@ -0,0 +1,10 @@
+#
+# Adds extensions with multiple dependencies to VMSS using 'provisionAfterExtensions' property and validates they are
+# enabled in order of dependencies.
+# NOTE: This scenario is executed on an instance of a scaleset created by the combinator.
+#
+name: "ExtSequencing"
+tests:
+  - "ext_sequencing/ext_sequencing.py"
+images: "random(endorsed)"
```

Comment:
Original DCR scenario runs on all endorsed images. 

I'm choosing to run the scenario on one random endorsed image because VMSS creation is time consuming and test execution cannot start until the combinator is done creating the VMSS resources and environments.